test/boolean_test.cpp

```diff
@@ -296,6 +296,67 @@ TEST(Boolean, SplitByPlane60) {
               splits.second.GetProperties().volume, 1e-5);
 }
 
+TEST(Manifold, ConvexConvexMinkowski) {
+  bool oldDeterministic = ManifoldParams().deterministic;
+  ManifoldParams().deterministic = true;
+  float offsetRadius = 0.1f;
+  float cubeWidth = 2.0f;
+  Manifold sphere = Manifold::Sphere(offsetRadius, 20);
+  Manifold cube = Manifold::Cube({cubeWidth, cubeWidth, cubeWidth});
+  Manifold sum = cube.MinkowskiSum(sphere);
+  EXPECT_NEAR(sum.GetProperties().volume,
+              powf(cubeWidth + (2 * offsetRadius), 3.f), 0.06f);
```

Comment:
Hmm, that looks like the volume of a cube, not a rounded cube. I think you need to add the volume of your sphere and subtract the volume of a cube that fits right around that sphere. Probably increase the offsetRadius to 1 as well.

Reply:
I think that adjustment to the formula only accounts for the 8 sphere corners; there are also the 8 cylindrical edges to account for... I'd really rather not work out the rounded cuboid volume for this 😅